template/src/App.js

```diff
@@ -1,18 +1,23 @@
-import logo from 'assets/logo.svg';
+import { Switch, BrowserRouter } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
 
-import useTranslation from './hooks/useTranslation';
-import './App.css';
+import RouteFromPath from './components/routes/RouteFromPath';
+import routes from './routes';
 
 function App() {
-  const t = useTranslation();
-
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>{t('home.welcomeMsg')}</p>
-      </header>
-    </div>
+    <>
+      <Helmet>
+        <title>RS React Redux Base</title>
+      </Helmet>
+      <BrowserRouter>
+        <Switch>
+          {routes.map((route, index) => (
+            <RouteFromPath key={`route${index}`} {...route} />
+          ))}
```

Comment:
What about implementing it like this? Using index to map elements is not a very good practice. Here maybe it's not a big deal because the routes are static but it sets a bad example.

```suggestion
          {routes.map((route) => (
            <RouteFromPath key={`route-${route.path}`} {...route} />
          ))}
```

Reply:
Good catch!